code/modules/paperwork/photography_journalism.dm

```diff
@@ -29,7 +29,17 @@
 			if(A.mind.prefs)
 				if(A.mind.prefs.economic_status == CLASS_UPPER)
 					sensational += 5
-
+					
+			// If this person is unimportant, we don't care if they were arrested
+			// If they're in an important role or upper class, check to see if they look arrested.
+			
+			if(sensational > 0) 
+				var/list/clothing_list = A.worn_clothing

```

Comment:
Here's a cleaner/less CPU intensive way of doing this. Each human (check under VV) has a variable called "w_uniform" which is always updated to their current worn uniform.  Also take note this code will not work if someone (somehow) changes the name of the orange jumpsuit, say, uses a hand labeller or something. This is where the istype proc would be useful.

**Code can look a bit like this:**

`if(A.w_uniform)
	var/obj/item/clothing/C = A.w_uniform
	if (istype(C, /obj/item/clothing/under/color/orange))
		sensational = sensational * 2
`

Reply:
Updated with the new code!